tests/snappi_tests/variables.py

```diff
@@ -66,3 +69,160 @@
                      5: 0,
                      6: 1,
                      7: 0}
+
+
+def create_ip_list(value, count, mask=32, incr=0):
+    '''
+        Create a list of ips based on the count provided
+        Parameters:
+            value: start value of the list
+            count: number of ips required
+            mask: subnet mask for the ips to be created
+            incr: increment value of the ip
+    '''
+    if sys.version_info.major == 2:
+        value = unicode(value)          # noqa: F821
+
+    ip_list = [value]
+    for i in range(1, count):
+        if ip_address(value).version == 4:
+            incr1 = pow(2, (32 - int(mask))) + incr
+            value = (IPv4Address(value) + incr1).compressed
+        elif ip_address(value).version == 6:
+            if mask == 32:
+                mask = 64
+            incr1 = pow(2, (128 - int(mask))) + incr
+            value = (IPv6Address(value) + incr1).compressed
+        ip_list.append(value)
+
+    return ip_list
+
+
+def get_host_addresses(subnet, count):
+    try:
+        # Create an IPv4Network object
+        network = ipaddress.ip_network(subnet, strict=False)
+
+        # Generate all possible host addresses
+        all_hosts = list(network.hosts())
+
+        # Check if the requested count is within the available host range
+        if count > len(all_hosts):
+            raise ValueError("Requested count exceeds the number of available hosts in the subnet.")
+
+        # Return the list of host addresses up to the specified count
+        return all_hosts[:count]
+
+    except ValueError as e:
+        return str(e)
+
+
+ip = []
+peer_ip = []
+ipv6 = []
+peer_ipv6 = []
+# START ---------------------   T2 BGP Case -------------------
+'''
+    PRE-REQUISITE : The DUT ports must be Administratively Up and configured as Routed ports before starting the test
+'''
+# *********** Common variables for Performance and Outbound ****************
+T2_SNAPPI_AS_NUM = 65400
+T2_DUT_AS_NUM = 65100
+BGP_TYPE = 'ebgp'
```

Comment:
Can you add Fanout Device logic on the uplink linecard?

Reply:
implemented